src/core/base.jl

```diff
@@ -154,18 +143,16 @@ function build_sets(data :: Dict{AbstractString,Any})
     gen_idxs = collect(keys(gen_lookup))
     branch_idxs = collect(keys(branch_lookup))
 
-
-    buspair_indexes = collect(Set([(i,j) for (l,i,j) in arcs_from]))
-    buspairs = buspair_parameters(buspair_indexes, branch_lookup, bus_lookup)  
+    buspair_indexes = collect(Set([a[2:3] for a in arcs_from]))
+    buspairs = buspair_parameters(buspair_indexes, branch_lookup, bus_lookup)
```

Comment:
Sure, Lint.jl pointed out that `l` was defined but not used, so I tried to make it unnecessary. `[(i, j) for (NaN, i, j) in x] also works. @mlubin, is there any performance advantage to getting rid of `l`? I do think it looked better before my change.